modules/nf-core/nacho/normalise/meta.yml

```diff
@@ -0,0 +1,64 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: nacho_normalise
+description: |
+  NACHO (NAnostring quality Control dasHbOard) is developed for NanoString nCounter data.
+  NanoString nCounter data is a messenger-RNA/micro-RNA (mRNA/miRNA) expression assay and works with fluorescent barcodes.
+  Each barcode is assigned a mRNA/miRNA, which can be counted after bonding with its target.
+  As a result each count of a specific barcode represents the presence of its target mRNA/miRNA.
+keywords:
+  - nacho
+  - nanostring
+  - mRNA
+  - miRNA
+  - qc
+tools:
+  - NACHO:
+      description: |
+        R package that uses two main functions to summarize and visualize NanoString RCC files,
+        namely: `load_rcc()` and `visualise()`. It also includes a function `normalise()`, which (re)calculates
+        sample specific size factors and normalises the data.
+        For more information `vignette("NACHO")` and `vignette("NACHO-analysis")`
+      homepage: https://github.com/mcanouil/NACHO
+      documentation: https://cran.r-project.org/web/packages/NACHO/vignettes/NACHO.html
+      doi: "doi:10.1093/bioinformatics/btz647"
+      licence: GPL-3.0 license
```

Comment:
I think you should simply write `GPL-3.0` here. I don't know why the linter didn't complain. Please check with other modules.